pages/docs/tutorials/create-asyncapi-document.md

```diff
@@ -0,0 +1,162 @@
+
+Moving on, let's talk about the `servers` section.
+<CodeBlock>
+{`servers:
+  mosquitto:
+    url: mqtt://test.mosquitto.org
+    protocol: mqtt`}
+</CodeBlock> 
+
+In this section, you point to the Eclipse Mosquitto message broker. The `url` point to a real instance of the broker [hosted by the Mosquitto community](https://test.mosquitto.org/) and the `protocol` as MQTT. If you do not want to use the test instance, you can spin up your own broker locally with `docker run -it -p 1883:1883 eclipse-mosquitto:1.5`. But remember to change `url` to `mqtt://localhost`
+
+Now lets move on to the `channels` section. This section is used to describe the event names your API will be publishing and/or subscribing to.
+
+<CodeBlock>
+{`channels:
+  light/measured:
+    publish:
+      summary: Inform about environmental lighting conditions for a particular streetlight.
+      operationId: onLightMeasured`}
+</CodeBlock>
+
+In this example, `light/measured` is the channel name the Streetlight API will `subscribe` to (i.e, to interact with the Streetlight API you `publish` to the broker). The `operationId` property, describes what is the name of function or method that takes care of this functionality in the generated code.
+
+Next is the `payload` property which is used to understand how the event should look like when publishing to that channel:
+
+<CodeBlock>
+{`      payload:
+        type: object
+        properties:
+          id:
+            type: integer
+            minimum: 0
+            description: Id of the streetlight.
+          lumens:
+            type: integer
+            minimum: 0
+            description: Light intensity measured in lumens.
+          sentAt:
+            type: string
+            format: date-time
+            description: Date and time when the message was sent.`}
+</CodeBlock>
+
+The `payload` property defines the content of the event using AsyncAPI schemas. It means that your event payload should contain an `id` and a `lumens` property —which are integers bigger than zero—, and a `sentAt` property that should be a string containing a date and time.
+
+>  JSON Schema Draft 07 is 100% compatible with AsyncAPI schemas. You can also use other standards to describe payload schema, like, for example [Avro](https://github.com/asyncapi/avro-schema-parser#usage).
+
+## Summary
+
+In this tutorial, you learned how to create an AsyncAPI specification document via a real-life example with an IoT use case.
+
+This tutorial is only a bootstrap; you'll need to add your own business logic into it. Take some time to play with it. There are still lots of things to be covered, but the intent of this tutorial is to make it simple for you to get an idea of the potential.
+
+## Next steps
+Now that you've completed this tutorial, go ahead to learn how to [validate your AsyncAPI document with Studio](https://www.asyncapi.com/docs/tutorials/studio-document-validation).
```

Comment:
```suggestion
Now that you've completed this tutorial, go ahead to learn how to [validate your AsyncAPI document with Studio](/docs/tutorials/studio-document-validation).
```

Also, I think @alequetzalli, you have to make a special dedicated PR to align the `Next Steps` of each page in Docs, after merging all the Doc pages of gsod :sweat_smile:. I'm very much confused about the sequence in which pages will be connected, but hope so, it is very much clear to you at the end :smile:.

Reply:
they are all ready, i already fixed them :P :)

Reply:
I don't think you pushed changed

Reply:
I mean pushed changed related to the link `https://www.asyncapi.com/docs/tutorials/studio-document-validation`